### PR TITLE
Refactor/replace shuffle commercial provider script

### PR DESCRIPTION
**Pull request by drupol**
This new PR:

- Originates from #1052
- Follows #1056

I have to admit that developing the least thing in this project is a nightmare, there's a process which is continuously building SVGs, it never stops. Rendering a simple page takes more than a minute.
I tried to find the root issue causing this, but I couldn't find anything. I'm pretty sure there are room for improvements.

### CODE REVIEW

**Review by fricklerhandwerk — APPROVED** [review]
Merging this as it has been extensively reviewed. We can fix forward if there are still issues. @Garbas would be great if we could prune the commit log. I'm not sure if I have push access to master, but if you give green light I'll do it.

**Comment by drupol** [comment]
Deployed ! https://649b2dff18942208dba65096--nixos-homepage.netlify.app/community/commercial-support.html

File: community/commercial-support.tt
Comment (Mic92) [inline community/commercial-support.tt:168]: This does not look correct, fixed in https://github.com/NixOS/nixos-homepage/pull/1140

**Pull request merged**
